core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorBodyDirective.js

```diff
@@ -64,6 +85,8 @@ oppia.directive('simpleEditorBody', [
             return false;
           };
 
+
```

Comment:
Drop these two blank lines please. (In the future, look at the diff on [Files Changed](https://github.com/oppia/oppia/pull/3509/files) to ensure that the changes are exactly what you want them to be.)